### ISSUE TITLE
pfe-popover

### ISSUE THREAD

**Issue by heyMP** [patternfly/patternfly-elements#1407]
# Component name: 

> General description of the component goal.

![Screen Shot 2021-02-25 at 10 51 39 AM](https://user-images.githubusercontent.com/3428964/109179038-7db7f500-7757-11eb-9a06-c80af2742329.png)

![Screen Shot 2021-02-25 at 10 51 43 AM](https://user-images.githubusercontent.com/3428964/109179235-afc95700-7757-11eb-81c9-1ea7a8e3ae2b.png)

**Comment by heyMP** [comment]
@coreyvickery pfe-popover is ready for initial design review.

https://deploy-preview-1406--happy-galileo-ea79c4.netlify.app/elements/pfe-popover/demo/

**Comment by heyMP** [comment]
The "X" is coming from a pfe-icon set.  It's called "web-icon-close".

![Screen Shot 2021-03-15 at 11 49 34 AM](https://user-images.githubusercontent.com/3428964/111181764-ac6e0200-8584-11eb-8833-a4ff7b4f1a7b.png)
https://deploy-preview-1406--happy-galileo-ea79c4.netlify.app/elements/pfe-icon/demo/

**Comment by coreyvickery** [comment]
@heyMP Okay thanks, maybe one day we'll use the same icons as PatternFly. Let me know when a final review is ready.

**Comment by heyMP** [comment]
@coreyvickery what I can do is embed the PatternFly icons into the component until we come up with a plan for moving them into pfe-icon.

**Comment by coreyvickery** [comment]
@heyMP That would be great, let me know when you have another link I can review.

**Comment by heyMP** [comment]
@coreyvickery ready for followup review. https://heymp-pfe-gemini.surge.sh/elements/pfe-popover/demo/

- [x] The headline font should be Display, not Regular (Changed to Display. Is the font-weight correct (currently 500)?)
- [x] Where are we pulling the X from? (I embedded the icon from Patternfly directly into the component for now.)
- [x] Spacing in between the buttons is 24px

**Comment by coreyvickery** [comment]
@heyMP Thanks for sending, just a few more tiny fixes and then we're done.

### Close button
* Hover/active state for the light theme close button is #151515; dark theme is #D2D2D2.

### Primary text button
* Normal state for the background is #2B9AF3; hover/active state is #73BCF7.
* Text always stays white.
* Distance between buttons is 16px.

<img width="202" alt="Screen Shot 2021-04-15 at 3 28 23 PM" src="https://user-images.githubusercontent.com/45767595/114927218-3cf76680-9dff-11eb-83b6-46c6a56c15d6.png">

### Secondary (ghost) text button
* Normal state for the border is #2B9AF3; hover/active is the same color and the same border thickness value as the light theme button border.
* Text always stays white.
* Distance between buttons is 16px.

<img width="248" alt="Screen Shot 2021-04-15 at 3 28 01 PM" src="https://user-images.githubusercontent.com/45767595/114927239-41bc1a80-9dff-11eb-8ac2-9f3426eabc37.png">

**Comment by heyMP** [comment]
Hey, @coreyvickery here are some additional updates. 😁

https://heymp-pfe-gemini.surge.sh/elements/pfe-popover/demo/

- [x] Hover/active state for the light theme close button is #151515; dark theme is #D2D2D2.
- [x] Distance between buttons is 16px. (in the future, we might want to break this functionality out into its own component. Maybe something like pfe-button-group.)
- [ ]  Primary & Secondary color changes for normal and hover state. (I'm wondering if this work needs to be within pfe-button.  I'm not sure that we would want pfe-popover to be responsible for controlling color changes for light mode and dark mode on pfe-button.  Does that sound right to you?)

**Comment by coreyvickery** [comment]
@heyMP I agree about all things pfe-button, I'm not sure of the following either just yet, but I'll try and track down more information.

* If pfe-button has specs for groups
* If pfe-button has specs for dark theme

Final changes and then you can close this issue.

* Dark theme close button normal/link state is white
* Dark theme close button hover/active state is #D2D2D2

**Comment by heyMP** [comment]
@coreyvickery hot changes 🔥

https://deploy-preview-1406--patternfly-elements.netlify.app/elements/pfe-popover/demo/

- [x] dark close button white, button hover/active #D2D2D2

**Comment by coreyvickery** [comment]
@heyMP Perfect, ready to close!
